src/api/json_tokenize.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef MIGRAPHX_GUARD_API_RTGLIB_JSONIZE_ATTR_STRING_HPP
+#define MIGRAPHX_GUARD_API_RTGLIB_JSONIZE_ATTR_STRING_HPP
+
+#include <string>
+#include <migraphx/config.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+
+using siter = std::string::const_iterator;
+using token = std::pair<siter, siter>;
+
+std::string json_tokenize(const std::string& s);
```

Comment:
This should probably be called something like `convert_json`, as tokenize implies we would be returning tokens.

Reply:
Maybe we can unit tests for this function(without the constructor API):

```cpp
EXPECT(convert_json("{key: 1}") == "{\"key\": 1}");

Reply:
Looks like we do not expose the api convert_to_json()